examples/helloworld.cf

```diff
@@ -31,7 +31,20 @@ body common control
 
 bundle agent hello
 {
+  files:
+      "/tmp/hello_world"
+        create => "true",
+        content => "Hello world!$(const.n)";
+
   reports:
+      "/tmp/hello_world"
+        printfile => cat( $(this.promiser) );
+}
 
-      "Hello world!";
+body printfile cat(file)
+# @brief Report the contents of a file
+# @param file The full path of the file to report
+{
+      file_to_print => "$(file)";
+      number_of_lines => "inf";
```

Comment:
As mentioned in slack, I still think this is too much code for a hello world example, mine would be:

```
bundle agent main
{
  files:
    "/tmp/hello_world"
      create => "true",
      content => "Hello, CFEngine!$(const.n)";
}
```

(removing the `create` once we make that change).

Reply:
Thanks! That looks simpler :)